docs/clustering.md

```diff
@@ -259,6 +287,29 @@ This also relies on shared storage (such as NFS) being hosted at the following a
 
 Adjust the 3 pre_tasks `Mount /var/lib/pulp` as needed for your shared storage.
 
+Install the `nfs-utils` package:
+```
+# yum install nfs-utils
+```
+Mount the NFS share:
+
+* use the `showmount` utility to discover the shares from NFS server:
+```
+$ showmount --exports <my-nfs-server-address>
+```
+* mount the `/var/lib/pulp` and `/var/lib/pulp/pulpcore_static` [share points](https://github.com/pulp/pulp_installer/blob/main/roles/pulp_webserver/README.md#role-variables):
+```
+# mount <my-nfs-server-address>:<exported-file-system-for-pulp-user-home> /var/lib/pulp
+# mount <my-nfs-server-address>:<exported-file-system-for-webserver-static-dir> /var/lib/pulp/pulpcore_static
```

Comment:
With the change I made in #1206, it is not sufficient to merely mount the shares. They must be in fstab.